mobilecoind-json/tests/integration_test.py

```diff
@@ -57,8 +57,20 @@ def get_monitor(self, monitor_id):
     def get_ledger_local(self):
         return self.request(f"ledger/local")
 
-    def account_key_from_mnemonic(self, mnemonic):
-        return self.request("account-key-from-mnemonic", {"mnemonic": mnemonic})
+    # Takes the json loaded from a private key file, tests if it is mnemnonic or root entropy
+    #
+    # If it is mnemonic, we pass the mnemonic to `account-key-from-mnemonic` route
+    # Otherwise, we hit the `entropy/{root_entropy}` route.
+    # Both return Json account key response.
+    def account_key_from_json(self, obj):
```

Comment:
Could be renamed to json_obj maybe? But otherwise no complaints.

Reply:
yah in the interest of not making a change in a cherry-pick PR i'm gonna skip that for now